packages/minifier-js/package.js

```diff
@@ -1,26 +1,10 @@
 Package.describe({
   summary: "JavaScript minifier",
-  version: "1.2.18"
-});
-
-Npm.depends({
-  "uglify-js": "2.7.5"
-});
-
-Npm.strip({
-  "uglify-js": ["test/"]
+  version: "2.0.0"
 });
 
 Package.onUse(function (api) {
-  api.export(['UglifyJSMinify', 'UglifyJS']);
+  api.use('babel-compiler');
+  api.export(['meteorBabelMinify']);
```

Comment:
It may make sense to keep this as `BabelMinify` for consistency and since it's a global variable.  No need to change right now, I'm just leaving this as a note for another reviewer. :wink: